examples/advanced/config_search_path/dataset/cifar10.yaml

```diff
@@ -0,0 +1,2 @@
+name: cifar10
```

Comment:
put the extra configs in a second directory next to conf, e.g:
`additional_conf`, and add pkg://additional_conf to hydra.searchpath.

Reply:
fixed.